fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -506,13 +509,43 @@ public Response createObject(@HeaderParam("Content-Disposition") final ContentDi
             session.save();
         } catch (final RepositoryException e) {
             throw new RepositoryRuntimeException(e);
+        } catch (final Exception e) {
+            checkForInsufficientStorageException(e, e);
         }
 
         LOGGER.debug("Finished creating resource with path: {}", newObjectPath);
         return createUpdateResponse(resource, true);
     }
 
     /**
+     * @param rootThrowable The original throwable
+     * @param throwable The throwable under direct scrutiny.
+     * @throws RepositoryRuntimeException
+     * @throws InsufficientStorageException
```

Comment:
It is not necessary to include runtime exceptions in javadocs (RepositoryRuntimeException and InsufficientStorageException)